doc/content/modules/user-manual/pages/release-notes/2025.2.0.adoc

```diff
@@ -130,6 +130,8 @@ image::release-notes-diagrams-export-as-png.png[Export diagram as PNG, width=50%
 - Display in the _Details_ view the properties _isDefault_ and _isInitial_ in Core tab of `FeatureValue` concept.
 
 
+- The direct edit tool on `Feature` elements now allows to use the ":=" and "default" symboles.
```

Comment:
symboles => symbols

Reply:
Please also add a reference to the documentation section `user-manual/hands-on/how-tos/model-management` (which need to be updated, see my first comment).